### PR TITLE
1911 workers molecule3

### LINKED ISSUE #1911: add workers molecule3

### PR DESCRIPTION
**Pull request by kayiwa**
- remove rails-app
- add sidekiq_workers tests and role to molecule3
- add sneakers_workers tests and role to molecule3
closes #1911

### CODE REVIEW

File: roles/sidekiq_worker/molecule/default/verify.yml [hunk 1]
```diff
@@ -3,18 +3,16 @@
...
-    assert:
-      that:
-        - not pkg_status.changed
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sidekiq-workers status"
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sidekiq-workers restart"
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sidekiq-workers start"
+    check_mode: true
+    register: present
+    failed_when:
+      - present is changed
```
Comment (carolyncole) [inline roles/sidekiq_worker/molecule/default/verify.yml:18]: The old test used to check for `f = host.file('/etc/systemd/system/sidekiq-workers.service')` Is this still needed?

File: roles/sneakers_worker/molecule/default/verify.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      line: "{{ item }}"
+      state: present
+    loop:
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sneakers status"
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sneakers restart"
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sneakers start"
+    check_mode: true
+    register: present
+    failed_when:
+      - present is changed
```
Comment (carolyncole) [inline roles/sneakers_worker/molecule/default/verify.yml:18]: The old tests used to check for `f = host.file('/etc/systemd/system/sneakers.service')` Is this still needed?  I see the tests for sudoers which look great, but the service itself might not exist.

**Review by carolyncole — CHANGES_REQUESTED** [review]

File: roles/sneakers_worker/molecule/default/verify.yml (hunk 2 above)
Comment (kayiwa) [reply roles/sneakers_worker/molecule/default/verify.yml:18]: these are systemd tests that never ran before molecule3. I elected not to introduce them and issued them with #1926

Comment (kayiwa) [reply roles/sneakers_worker/molecule/default/verify.yml:18]: OK?

**Review by carolyncole — APPROVED** [review]

**Pull request merged**